client_admin/pulp/client/admin/tasks.py

```diff
@@ -271,6 +290,36 @@ def retrieve_tasks(self, **kwargs):
                 filters={'state': {'$in': ['running', 'waiting']}}, fields=self.FIELDS)
         return tasks
 
+    def purge(self, **kwargs):
+        """
+        Attempts to purge completed tasks with states finished, error
+        and/or skipped. If a state does not support purging, a bad request(400)
+        will be raised from the server. If no states is mentioned, it raises the
+        CommandUsage Exception.
+        :return:    Exception or Success message from REST API call.
+        :rtype:     PulpServerException or CommandUsage Exception
+        """
+        self.context.prompt.render_title('Purge Completed Tasks')
+
+        if kwargs.get(self.purge_all.keyword) and kwargs.get(self.purge_state.keyword):
+            msg = _('These arguments cannot be used together')
+            self.context.prompt.render_failure_message(msg)
+            return
+
+        if kwargs.get(self.purge_state.keyword):
+            states = kwargs[self.purge_state.keyword]
+        elif kwargs.get(self.purge_all.keyword):
+            states = VALID_STATES
+        else:
+            raise CommandUsage
```

Comment:
@shubham90 as a random example, this is how we document raised exceptions https://github.com/pulp/pulp/blob/master/server/pulp/server/webservices/views/repositories.py#L447